docs/source/notes/autograd.rst

```diff
@@ -91,6 +91,8 @@ the module level with :meth:`nn.Module.requires_grad_()`.
 When applied to a module, ``.requires_grad_()`` takes effect on all
 of the module's parameters (which have ``requires_grad=True`` by default).
 
+Tensors that are not module parameters have ``requires_grad=False` by default.
+
```

Comment:
Note that the formatting above is missing the second backquote after the requires_grad

Reply:
Thanks, but now moot.

BTW, I see some lines like :meth:`nn.Module.requires_grad_()`. Do you know why those are able to get away with single backquotes?

Reply:
The ones with a marker before like `:meth:` or `:attr:` need only single backquotes.
The inline code is marked by the double backquotes.